tests/integration/targets/ses_identity/tasks/main.yaml

```diff
@@ -370,6 +370,53 @@
           identity: "{{ email_identity }}"
           state: absent
   # ============================================================
+  - name: test clear notification settings
+    block:
+      - name: test topic
+        sns_topic:
+          name: "{{ notification_queue_name }}-{{ item }}"
+          state: present
+        register: topic_info
+        with_items:
+          - bounce
+          - complaint
+          - delivery
+      - name: register email identity
+        aws_ses_identity:
+          identity: "{{ email_identity }}"
+          state: present
+          bounce_notifications:
+            topic: "{{ topic_info.results[0].sns_arn }}"
+          complaint_notifications:
+            topic: "{{ topic_info.results[1].sns_arn }}"
+          delivery_notifications:
+            topic: "{{ topic_info.results[2].sns_arn }}"
+      - name: clear notification settings
+        aws_ses_identity:
+          identity: "{{ email_identity }}"
+          state: present
```

Comment:
Generally, not passing the parameter shouldn't cause any changes.

It should be more a case of explicitly passing something like `bounce_notifications: {}`

Reply:
Glad to help 🙌  I think what you're saying makes sense. I pushed a change to not cause any modification when the parameter is not present.

Do you think that we should now also update the documentation to reflect that? I think it is diverging.
https://github.com/ansible-collections/community.aws/blob/86c60b4957a2baa0bc8b00ef1ed9adf90a56ed9c/plugins/modules/ses_identity.py#L37

Reply:
Yes, those docs definitely need updating.omitting `bounce_notifications` shouldn't result in a change.  Adding `bounce_notifications`, but omitting `topic` should remove the configuration.

What might also be good would be making `topic` required if `include_headers` is set.  (done in the argument_spec, but I think we should do that as a separate change).  I'll follow up once we merge this and tag you.

Reply:
Amazing, thanks.